### DISCUSSION TITLE
I can help you out with getting MacOS for dev

### DISCUSSION THREAD

**Discussion by 3li-jpg** [Shabinder/SpotiFlyer#74] [Ideas]
HI 
I saw that you couldn't afford to buy a MacBook for dev. I would recommend that you follow this guide on how to dual boot macOS, so that u can continue with developing an app for IOS.  This method is popularly used by developers who want more performance on MacOs for lesser money. It is completely stable for use. I'm gonna leave the link below for the guide.

https://dortania.github.io/OpenCore-Install-Guide/

If you encounter any issue or need any help with this guide, you may please join discord.gg/hackintosh , the community is very friendly 
Thank You

**Comment by Shabinder** [comment]
@BankruptKat  Thanks for reaching out , I have already tried various macOS versions like Mojave/High Seirra,etc and on My laptop(Toshiba Sattelite C850 i3-2348m) have various issues because of which I couldn't develop IOS App: 
 - Brightness is always **Max** (tried various fixes like ssdt changes, IntelBacklightInjector,etc, **none worked** ) so cant use laptop with such high brightness
 - Xcode was very very slow(My laptop's old hardware at fault here)

I have started saving money to buy a laptop(maybe USED becuz of low cost),still a long road ahead to gather req. amount.

**Reply by 3li-jpg** [reply]
Hi
Oh then I would recommend getting a laptop that has a core I5 (quad core ones) and a minimum of 16GB ram and an SSD if u are getting a brand new laptop, MacBooks are not worth paying extra for MacOS when dual-booting macOS on a Windows/Linux system has become more efficient and stable, I am running a core i5 6th gen laptop CPU and 20GB ram (on HDD), and macOS runs without any hiccups, and I would recommend Newegg if u are looking for laptops instead of local shops which charge more, Also another great way would be custom building a pc , Since you would have the option to upgrade later on , I have included a pc part picker list to a decent pc for Rs. 30K , i don't think you could any of these parts cheaper in local shops 
https://in.pcpartpicker.com/list/gh3wJf
If u need a pc parts list that is cheaper , do let me know your budget and i can help you out
Please do let me know if you want any more help
Thank You

**Reply by Shabinder** [reply]
What is your laptop model?
I guess macOs support varies greatly as per hardware , my budget is 50k , cab you suggest some laptops?

**Reply by 3li-jpg** [reply]
From digging around a bit , i found some perfect laptops for ur budget , i will list the links to the laptops

https://www.newegg.com/global/in-en/black-gateway-gwtn141-mainstream/p/1TS-000C-000W4 (very good price to performance , has 16gb ram and 512gb ssd , from a branch of acer laptops)

https://www.newegg.com/global/in-en/p/1TS-0016-01Z85 ( has 512gb ssd but only 8gb ram , but an approx 1 inch bigger screen than the first one , backlit keyboard) 

https://www.newegg.com/global/in-en/pure-silver-acer-aspire-5-a515-55-576h-mainstream/p/N82E16834316932?&quicklink=true ( everything is the same as the msi laptop) 

However if u would want to do some video editing or stuff like that , i would recommend a desktop which would be way more powerful than laptops and would cost the same 
Please do let me know if u need any other help

**Reply by Shabinder** [reply]
Hey , thanks for your suggestions , I have made my parents agree to go somewhat out of range so ,now I am looking at Hp Pavillion Gaming Laptop , 

One is with ryzen 5-3550H and one is intel i5-9300H

I will upgrade ram from 8gb to 16 gb myself.

So which one should be more compatible with hackintoshing , ryzen 5 or intel i5?

**Reply by Shabinder** [reply]
after some researching I have come to known, 
MacOs wont support Ryzen igpu(big drawback as Nvidia wont work too) whereas it supports Intel igpu.
MacOS will benefit from QuickSync feature from Intel Cpu(+1 for intel)

So , it seems intel is better option.

**Reply by 3li-jpg** [reply]
Hey 
I would recommend that you get a core i5 laptop if u want an easy installation , but Ryzen 5 laptops also support hackintosh just that there is a bit more process than intel based systems since apple officially used intel chips on macs.
Another thing i would recommend is that since you are getting a new laptop , get a 10th gen 10300H instead of 9300H for a slight performance Increase for not so much more . I will leave links to some gaming laptops under 70K that would be perfect for hackintosh as well as good gaming experience .
https://www.amazon.in/ASUS-GeForce-Graphics-Fortress-FX566LI-HN026T/dp/B08CY4WKM4/ref=mp_s_a_1_3?dchild=1&keywords=gaming+laptop+10th+gen&qid=1616599364&s=computers&sr=1-3 (144hz display , gtx 1650ti , 256gb ssd + 1tb hdd, core i5 10300H)
https://www.amazon.in/Lenovo-15-6-inch-Windows10-Graphics-81Y400VBIN/dp/B08SK1Z5CK/ref=mp_s_a_1_29?dchild=1&keywords=gaming+laptop+10th+gen+gtx&qid=1616600054&s=computers&sr=1-29 ( literally everything the same as mentioned above ) 
I would highly highly recommend spending some extra money for these kinds of performance gains
